### ISSUE TITLE
Not receiving email for `pod trunk register`

### ISSUE THREAD

**Issue by scannillo** [CocoaPods/cocoapods-trunk#179]
Hello,

I am a CocoaPods creator and am unable to successfully register my email on my new machine. No email is ever sent to me after running this:

```
pod trunk register scannillo@paypal.com 'Samantha Cannillo'
```

I greatly appreciate your help into getting this resolved! Please let me know if there is any additional information I can provide.

**Comment by scannillo** [comment]
👋 Hello - we are still having this issue and are unable to make necessary updates to our packages.

**Comment by scannillo** [comment]
Closing in favor of - https://github.com/CocoaPods/CocoaPods/issues/11850

**Comment by orta** [comment]
I've unblocked this email! Sorry, don't have much time for CocoaPods admin anymore

**Comment by scannillo** [comment]
Thank you @orta!
